File: Assignment_7/Question5.c
#include<stdio.h>

void Display(int iNo)
{
    if(iNo < 0)
    {
        iNo = -iNo;
    }

    int iCnt = 0;

    for(iCnt = iNo; iCnt >= iNo && iCnt <= (iNo*5); iCnt = iCnt + iNo)
    {
        printf("%d\t", iCnt);
    }

}

int main()
{
    int iValue = 0;

    printf("Enter the number : \n");
    scanf("%d", &iValue);

    Display(iValue);


    return 0;
}

File: Assignment__21/Question5.java
//Write a program to print all numbers from 1 to N that are divisible by both 2 and 3.

import java.util.Scanner;

class Function
{
    void Display(int iNo)
    {
        int iCnt = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if(iCnt % 2 == 0 && iCnt % 3 == 0)
            {
                System.out.println(iCnt +" ");
            }
        }

    }
}

class Question5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        Function fobj = new Function();

        fobj.Display(iValue);

    }

}
